# HG changeset patch
# User poonam
# Date 1362616238 28800
# Node ID ef36be1ae9bc7d061721f8890797e5d89c901846
# Parent  cf2b7f0f7a2188a1421bdd033e88bf4b0aa6870c
8006309: More reliable control panel operation
Summary: Added a comment in the dead Kernel code
Reviewed-by: ahgross, sla, coleenp

diff --git a/src/share/vm/runtime/thread.cpp b/src/share/vm/runtime/thread.cpp
--- hotspot/src/share/vm/runtime/thread.cpp
+++ hotspot/src/share/vm/runtime/thread.cpp
@@ -3416,6 +3416,9 @@
       os::dll_build_name(buffer, sizeof(buffer), Arguments::get_dll_dir(), name);
       library = os::dll_load(buffer, ebuf, sizeof ebuf);
 #ifdef KERNEL
+
+      // Dead code, KERNEL is never built in JDK7 or later. This code will be removed in a future update release.
+
       // Download instrument dll
       if (library == NULL && strcmp(name, "instrument") == 0) {
         char *props = Arguments::get_kernel_properties();
